revolution_16/sql/revolution_16.sql

```diff
@@ -9,13 +9,16 @@ INSERT INTO access (access_id, access_title) VALUES(1, 'User');
 INSERT INTO access (access_id, access_title) VALUES(2, 'Moderator');
 INSERT INTO access (access_id, access_title) VALUES(3, 'Super Moderator');
 
-CREATE TABLE adminblock (
+CREATE TABLE block (
+  id int(11) NOT NULL AUTO_INCREMENT,
```

Comment:
id tinyint(3) UNSIGNED NOT NULL AUTO_INCREMENT,

Reply:
Généralement je place un int(11), mais ta proposition fonctionne également, de mémoire la table lblocks id tinyint(3) unsigned NOT NULL AUTO_INCREMENT !

Reply:
oui !  pourquoi mettre un int 11 alors que dans cette table on a aucune chance d'avoir 255 blocs donc au final on stock sur 1 byte au lieu de 4...  peux tu faire la correction...

Reply:
Correction faite !

Reply:
merci